media/css/custom.scss

```diff
@@ -0,0 +1,469 @@
+// Core variables and mixins
+@import "variables";
+@import "mixins";
+
+html,
+body {
+  height: 100%;
+  text-align: justify;
+  color: $gray;
+  font-size: 16px;
+  text-shadow: none;
+  background: $pale-cyan;
+}
+
+// Wrapper for page content to push down footer
+#wrap {
+  min-height: 100%;
+  height: auto !important;
+  // Negative indent footer by it's height
+  margin: 0 auto -60px;
+}
+
+.proper-content {
+  padding-top: $footer-height;
+  // 60px to make the container go all the way to the bottom of the topbar
+}
+
+// Set the fixed height of the footer here
+
+#push,
+footer {
+  height: $footer-height;
+}
+
+footer {
+  background-color: $lighter-gray;
+}
+
+// Lastly, apply responsive CSS fixes as necessary
+@media (max-width: 767px) {
+  footer {
+    margin-left: -20px;
+    margin-right: -20px;
+    padding-left: 20px;
+    padding-right: 20px;
+  }
+}
+
+.container {
+  .credit {
+    margin: 20px 0;
+  }
+}
+
+#feed-range {
+  cursor: pointer;
+  padding: 5px 10px;
+  border: 1px solid $white;
+  &:hover {
+    background-color: $widget-link-highlight;
+  }
+}
+
+#rssdata ul.rss-items {
+  display: none;
+  margin: 0;
+  padding: 0;
+  li {
+    display: block;
+    margin: 0;
+    padding: 0;
+  }
+  a {
+    margin: 0;
+    display: block;
+    font: 15px / 16px $font-stack;
+    padding: 5px;
+    background: $sidebar-background;
+    color: $white;
+    text-decoration: none;
+    border-bottom: 1px solid $sidebar-border-color;
+    &:hover {
+      background: $sidebar-link-highlight;
+      color: $white;
+      text-decoration: none;
+    }
+  }
+}
+
+#rsstitle {
+  padding: 6px;
+  color: $white;
+  font: 17px / 18px bold $font-stack;
+  text-decoration: none;
+  border-bottom: 2px solid $sidebar-border-color;
+  height: 25px;
+}
+
+#rssbottom {
+  padding: 2px 6px;
+  background: $sidebar-background;
+  color: $white;
+  text-decoration: none;
+  border-top: 2px solid $sidebar-border-color;
+  height: 20px;
+}
+
+#twitter ul.twits {
+  display: none;
+  margin: 0;
+  padding: 0;
+}
+
+#twitter_title {
+  padding: 6px;
```

Comment:
Properties should be ordered border-bottom, color, font, padding, text-decoration